controller/articles.controller.js

```diff
@@ -0,0 +1,21 @@
+const { fetchArticleById } = require('../models/articles.model');
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params
+
+ 
+  if (isNaN(article_id)) {
```

Comment:
I'd see what feedback psql gives you when you try to make the request with an invalid article id - could you make use of the response in a psql error handler?